_data/sites.json

```
@@ -3379,6 +3379,16 @@
             "delta.com"
         ]
     },
+    
+    {
+        "name": "DelugeRPG",
+        "url": "https://www.delugerpg.com/faq#delete",
+        "difficulty": "impossible",
```

```suggestion
        "difficulty": "hard",
```
According to the notes, it takes very long, but is not impossible.